scripts/west_commands/runners/nrf_common.py

```diff
@@ -0,0 +1,401 @@
+
+        # If there is nothing in the hex file for the network core,
+        # only the application core is programmed.
+        if not self.hex_refers_region(net_flash_start, net_flash_end):
+            self.op_program(self.hex_, erase_arg, qspi_erase_opt, defer=True,
+                            core='NRFDL_DEVICE_CORE_APPLICATION')
+        # If there is some content that addresses a region beyond the network
+        # core flash range, two hex files are generated and the two cores
+        # are programmed one by one.
+        elif self.hex_contents.minaddr() < net_flash_start or \
+             self.hex_contents.maxaddr() > net_flash_end:
+
+            net_hex, app_hex = IntelHex(), IntelHex()
+            for start, end in self.hex_contents.segments():
+                if net_flash_start <= start <= net_flash_end:
+                    net_hex.merge(self.hex_contents[start:end])
+                else:
+                    app_hex.merge(self.hex_contents[start:end])
+
+            hex_path = Path(self.hex_)
+            hex_dir, hex_name = hex_path.parent, hex_path.name
+
+            net_hex_file = os.fspath(
+                hex_dir / f'GENERATED_CP_NETWORK_{hex_name}')
+            app_hex_file = os.fspath(
+                hex_dir / f'GENERATED_CP_APPLICATION_{hex_name}')
+
+            self.logger.info(
+                f'{self.hex_} targets both nRF53 coprocessors; '
+                f'splitting it into: {net_hex_file} and {app_hex_file}')
+
+            net_hex.write_hex_file(net_hex_file)
+            app_hex.write_hex_file(app_hex_file)
+
+            self.op_program(net_hex_file, erase_arg, None, defer=True,
+                            core='NRFDL_DEVICE_CORE_NETWORK')
+            self.op_program(app_hex_file, erase_arg, qspi_erase_opt, defer=True,
+                            core='NRFDL_DEVICE_CORE_APPLICATION')
+        # Otherwise, only the network core is programmed.
+        else:
+            self.op_program(self.hex_, erase_arg, None, defer=True,
+                            core='NRFDL_DEVICE_CORE_NETWORK')
+
+    def reset_target(self):
+        if self.family == 'NRF52_FAMILY' and not self.softreset:
+            self.exec_op('pinreset-enable')
+
+        if self.softreset:
+            self.exec_op('reset', option="RESET_SYSTEM")
+        else:
+            self.exec_op('reset', option="RESET_PIN")
+
+    @abc.abstractmethod
+    def do_require(self):
+        ''' Ensure the tool is installed '''
+
+    def op_program(self, hex_file, erase, qspi_erase, defer=False, core=None):
+        args = {'firmware': {'file': hex_file, 'format': 'NRFDL_FW_INTEL_HEX'},
+                'chip_erase_mode': erase, 'verify': 'VERIFY_READ'}
+        if qspi_erase:
+            args['firmware']['qspi_erase_mode'] = qspi_erase
+        self.exec_op('program', defer, core, **args)
+
+    def exec_op(self, op, defer=False, core=None, **kwargs):
+        _op = f'{op}'
+        op = {'operation': {'type': _op}}
```

Comment:
`op = {'operation': {'type': f'{op}'}}`

Reply:
Sure, but why do you think it's better?

Reply:
Oh I see, I no longer have the need for `_op` later on! I used to in an earlier version! Thanks!